styles/tailwind.scss

```diff
@@ -0,0 +1,46 @@
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
+
+// Base
+@layer base {
+  * {
+    @apply text-gray-900;
+  }
+  h1, h2, h3, a {
+    @apply font-title;
+  }
+  p, div, span {
+    @apply font-body;
+  }
+  h1 {
+    @apply text-4xl md:text-5xl mb-3 font-bold;
+  }
+  h2 {
+    @apply text-3xl md:text-4xl mb-3 font-bold;
+  }
+  h3 {
+    @apply text-2xl md:text-3xl mb-3 font-semibold;
+  }
+  h4 {
+    @apply text-xl md:text-2xl;
+  }
+}
+
+
+// Components 
+@layer components {
+  .section {
+    @apply py-4 md:py-5 lg:py-8;
+  }
+
+  .container {
+    @apply mx-auto px-4 max-w-screen-xl;
+  }
+}
+
+// Utilities
+@layer utilities {
+
+}
```

Comment:
New line before EOF

Reply:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file